tests/device_tests/test_supershamir_reset_recovery.py

```diff
@@ -0,0 +1,286 @@
+import pytest
+
+from trezorlib import btc, device, messages
+from trezorlib.messages import ButtonRequestType as B, ResetDeviceBackupType
+from trezorlib.tools import parse_path
+
+
+@pytest.mark.skip_t1
+def test_reset_recovery(client):
+    mnemonics = reset(client)
+    address_before = btc.get_address(client, "Bitcoin", parse_path("44'/0'/0'/0/0"))
+    # TODO: more combinations
```

Comment:
That'd be nice :). Let's do it or note it somewhere (new issue? or one larger issue for everything).

Reply:
The only issue I see with this is that if we try to test all combinations it might add a lot of time to the test. It should probably be part of the larger issue.